spec/requests/providers/proceeding_merits_task/vary_order_controller_spec.rb

```diff
@@ -0,0 +1,150 @@
+require "rails_helper"
+
+RSpec.describe Providers::ProceedingMeritsTask::VaryOrderController do
+  let(:legal_aid_application) { create(:legal_aid_application, :with_proceedings, explicit_proceedings: %i[da001 se007]) }
+  let(:login_provider) { login_as legal_aid_application.provider }
+  let(:smtl) { create(:legal_framework_merits_task_list, :da001_and_se007, legal_aid_application:) }
+  let(:proceeding) { legal_aid_application.proceedings.find_by(ccms_code: "SE007") }
+
+  describe "GET /providers/applications/merits_task_list/:merits_task_list_id/vary_order" do
+    subject(:get_vary_order) { get providers_merits_task_list_vary_order_path(proceeding) }
+
+    before do
+      login_provider
+      get_vary_order
+    end
+
+    it "renders show with ok" do
+      expect(response).to have_http_status(:ok).and render_template("providers/proceeding_merits_task/vary_order/show")
+    end
+
+    context "when not authenticated" do
+      let(:login_provider) { nil }
+
+      it_behaves_like "a provider not authenticated"
+    end
+
+    context "when authenticated as a different provider" do
+      let(:login_provider) { login_as create(:provider) }
+
+      it_behaves_like "an authenticated provider from a different firm"
+    end
+  end
+
+  describe "PATCH /providers/applications/merits_task_list/:merits_task_list_id/vary_order" do
+    subject(:post_vary_order) do
```

Comment:
Even though I wish we were making a post request, we're not 😆 

Should we name this `patch_vary_order` to better reflect that?